srcpkgs/yazi/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'yazi'
+pkgname=yazi
+version=0.2.1
+revision=1
+build_style=cargo
+make_install_args="--path yazi-fm"
+hostmakedepends="pkg-config"
+makedepends="oniguruma-devel lua54-devel"
+depends="nerd-fonts-symbols-ttf"
+short_desc="Blazing fast terminal file manager"
```

Comment:
How about using this full description "Blazing fast terminal file manager written in Rust, based on async I/O"?